dqc_us_rules/dqc_us_0001.py

```diff
@@ -64,6 +156,49 @@ def filter_func(model_object):
                 )
 
 
+def _determine_namespace(val):
+    """
+    Determines which json file to use based off of the
+    namespace of the filing.
+
+    :param val: val from which to determine namespace
+    :type val: :class:'~arelle.ModelXbrl.ModelXbrl'
+    :return: Name of config file to use
+    :rtype: String
+    """
+    NS_2016 = "http://fasb.org/us-gaap/2016-01-31"
+    NS_2015 = "http://fasb.org/us-gaap/2015-01-31"
+    NS_2014 = "http://fasb.org/us-gaap/2014-01-31"
+    RESOURCE_DIR = 'resources'
+    RULE = 'DQC_US_0001'
+
+    if NS_2016 in val.modelXbrl.namespaceDocs.keys():
+        config_json_file = os.path.join(
+             os.path.dirname(__file__),
+             RESOURCE_DIR,
+             RULE,
+             'dqc_0001_2016.json')
```

Comment:
again, this is a nit, but is in our style guide, the closing param should be on the next line